src/App.js

```diff
@@ -1,13 +1,85 @@
 import React from 'react';
+import TodoList from './components/TodoComponents/TodoList';
+import TodoForm from './components/TodoComponents/TodoForm';
 
 class App extends React.Component {
   // you will need a place to store your state in this component.
   // design `App` to be the parent component of your application.
   // this component is going to take care of state, and any change handlers you need to work with your state
+  constructor() {
+    super();
+    this.state = {
+      todos: [
+        {
+          task: 'Organize Garage',
+          id: 1528817077286,
+          completed: false
+        },
+        {
+          task: 'Bake Cookies',
+          id: 1528817084358,
+          completed: false
+        }
+      ],
+      new: {},
+      // input: ''
+    }
+  }
+
+  newTodo = attr => {
+    const todo = {
+      task: attr.task,
+      id: Date.now(),
+      completed: false
+    }
+
+    return todo;
+  }
+
+  // changeHandler = e => {
+  //   this.setState({input: e.target.value})
+  //   console.log('input: ', this.state.input)
+  // }
+
+  addTodo = async todo => {
+    console.log('todo being added/task passed in: ', todo)
+    await this.setState({todos: [...this.state.todos, todo]});
+    console.log('todos: ', this.state.todos)
+  }
+
+  submitTodo = async (e, todo) => {
+    e.preventDefault();
+    const task = this.newTodo({
+      task: todo,
+    })
+    console.log('returned task: ', task)
+    await this.setState({new: task})
+    console.log('this.state.new: ', this.state.new)
+    this.addTodo(this.state.new)
+    this.setState({input: ''})
```

Comment:
You had the right idea of resetting the state of the input box to an empty string after submitting the new task. However, input is not defined in the state and the task name persists after submitting it. But very close